packages/twenty-front/src/modules/object-record/record-field/ui/meta-types/input/components/MultiSelectFieldInput.tsx

```diff
@@ -25,6 +28,12 @@ export const MultiSelectFieldInput = () => {
     onSubmit?.({ newValue: draftValue });
   };
 
+  const handleAddSelectOption = navigateToFieldOption
```

Comment:
Please write a classic handler, this ternary assignment is more difficult to read than a simple if statement